src/components/gallery/HdGalleryCarousel.vue

```diff
@@ -30,12 +30,16 @@
             :style="sizerStyles"
             class="gallery-carousel__item__sizer"
           />
-          <div
-            :style="{
-              'background-image': `url('${item.thumbnail}')`,
-            }"
-            class="gallery-carousel__item__background"
-          />
+
+          <picture v-if="usePictureElement" class="gallery-carousel__item__picture">
+              <source v-for="(source, media) in item.thumbnailPictureSources"
+                :key="media"
+                :media="`(${media})`" :srcset="source"
+              >
+            <img :src="item.thumbnail" :alt="item.caption" :srcset="item.thumbnailSrcSet">
+          </picture>
```

Comment:
Can't we just keep this block? As far as I've seen IE doesn't support `<picture>` but it would fallback to the `<img>`. Could you confirm it?

Reply:
It is correct. I've also tested in IE11, to validate. :) I think he's handling the backwards compatibility with that - based on the `if` condition. Probably can be consolidated though.

Reply:
Exactly

Reply:
I would give it a try first to see if we would need this fallback anyway (@aym3nb might help in here)